flexget/entry.py

```diff
@@ -290,3 +290,9 @@ def __hash__(self):
 
     def __repr__(self):
         return '<Entry(title=%s,state=%s)>' % (self['title'], self._state)
+
+    def to_dict(self):
```

Comment:
Nah, I guess not. I just wasn't sure how to get it done, jsonify was giving me a hard time. I'll change this